tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/binaryoperations/IntegerRangeOperatorTest.java

```diff
@@ -188,6 +188,16 @@ public void testIntRangeWithHexIntLiterals() {
         BRunUtil.invoke(result, "testIntRangeWithHexIntLiterals");
     }
 
```

Comment:
Isn't this test file for range expressions? The tests for this should probably go in int lang lib tests and foreach/from tests.